commands/template_store_list.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) OpenFaaS Author(s) 2018. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	// DefaultTemplatesStore is the URL where the official store can be found
+	DefaultTemplatesStore = "https://raw.githubusercontent.com/openfaas/store/master/templates.json"
+)
+
+var (
+	templateStoreURL string
+)
+
+func init() {
+	templateStoreListCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "Shows additional language and platform")
+	templateStoreListCmd.PersistentFlags().StringVarP(&templateStoreURL, "url", "u", DefaultTemplatesStore, "Use as alternative store for templates")
+
+	templateStoreCmd.AddCommand(templateStoreListCmd)
+}
+
+// templateStoreListCmd lists templates from default store or custom store if set
+var templateStoreListCmd = &cobra.Command{
+	Use:     `list`,
+	Short:   `List templates from OpenFaaS organizations`,
+	Aliases: []string{"ls"},
+	Long:    `List templates from official store or from custom URL`,
+	Example: `  faas-cli template store list
+  faas-cli template store ls
```

Comment:
@johnmccabe should we mention the ENV-VAR in the command help text which can be used to override the template store URL?

Reply:
Mentioned it in `Long` explanation not in `Example` ✔️